src/main/java/Strings/PleaseDeleteMe.java

```diff
@@ -0,0 +1,53 @@
+package Strings;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PleaseDeleteMe {
+
+    String name ="ray maldoNADo ESTrada";
```

Comment:
Since this is the firstName, are we followign naming conventions for firstName to depict we are in fact calling a person's name rather a generic name varible which can be related to any name?